### PR TITLE
Add outbound to MessageSender

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
currently we're only saving inbound messages, we need be send out too via 'outbound'

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review

File: lib/message_sender.js [hunk 1]
```diff
@@ -37,6 +37,30 @@ MessageSender.prototype = {
...
+      Gets passed through directly to service. Consult service docs/protocol.
+      Example object content:
+            {
+                "to_addr": "+27123",
+                "identity": identity uuid,
+                "content": "my outbound message",
+                "metadata": {}
+            }
+    */
+    create_outbound_message: function(outbound_msg_info) {
```
Comment (imsickofmaps) [inline lib/message_sender.js]: please make it params again with this one. `metadata` can be none/undefined on but if so should be set to `{}` before posting.

**Comment by imsickofmaps** [comment]
Done, change to params, add a test for metadata not supplied, then I'll review and 👍

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps Another review please

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**